src/components/RightPane/CoursePane/SearchForm/PrivacyPolicyBanner.tsx

```diff
@@ -20,8 +20,10 @@ class PrivacyPolicyBanner extends PureComponent<PrivacyPolicyBannerProps> {
             <Paper variant="outlined" className={this.props.classes.container}>
                 <Typography variant="body2">
                     We use cookies to analyze website traffic and track usage, with the aim of improving your experience
-                    on AntAlmanac. By continuing to use this website, you are giving consent to store Google Analytics
-                    cookies on your device.
+                    on AntAlmanac. By continuing to use this website, consent to our{' '}
+                    <a href={'https://github.com/icssc/AntAlmanac/blob/privacy-policy/PRIVACY-POLICY.md'}>
```

Comment:
```suggestion
                    <a href={'https://github.com/icssc/AntAlmanac/blob/main/PRIVACY-POLICY.md'}>
```
Since we're planning to push this to main, feel free to update the link ahead of time.